src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -145,7 +145,12 @@ export default class DropDownComponent extends ComponentBase<azdata.DropDownProp
 			}
 			this._editableDropdown.enabled = this.enabled;
 			this._editableDropdown.fireOnTextChange = this.fireOnTextChange;
-			this._editableDropdown.input.setTooltip(!this.enabled ? this._editableDropdown.input.value : '');
+
+			if (!this.enabled) {
+				this._editableDropdown.input.setTooltip(this._editableDropdown.input.value);
+			} else if (this._editableDropdown.input.inputElement.title) {
```

Comment:
What scenario is this fixing? Because it seems to me that what was there before should cover the scenario you wanted - when the input is disabled then it will set the tooltip to the value. And all other times it'd clear the tooltip.

Is your change purely to avoid calling setTooltip as often? If so I don't really think this is worth the call - it's pretty cheap and I'd be slightly worried of there being an edge case where we didn't clear it correctly.

Reply:
yes, the purpose of this change is to avoid calling setTooltip every time a dropdown property is updated.

when the dropdown is disabled, add the current value to input.title (tooltip)
when the dropdown is enabled and has a non-empty tooltipinput.title, remove the input.title

Reply:
Was there a bug you were seeing with that? I think unless there's an actual issue you saw then it'd be best to just leave it as it is for now.

Reply:
there was no bug. I reverted back to 
`this._editableDropdown.input.setTooltip(!this.enabled ? this._editableDropdown.input.value : '');`